### ISSUE TITLE
Fix `npm install` warnings

### ISSUE THREAD

**Issue by mozfreddyb** [mozilla/srihash.org#97]
This could be a good bug for a contributor. :)

```
npm WARN package.json srihash.org@0.1.0 license should be a valid SPDX license expression
npm WARN deprecated lodash@0.9.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0
npm WARN deprecated lodash@2.4.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0
npm WARN optional dep failed, continuing fsevents@1.0.7
npm WARN prefer global grunt-cli@0.1.13 should be installed with -g
grunt-copyright@0.2.0 node_modules/grunt-copyright
```

**Comment by XhmikosR** [comment]
These are already obsolete (there are many more now) and the only valid warning is actually the license one which doesn't show in Travis since it's using an ancient npm.

Is there any reason why you keep using node.js 0.10 in Travis?

**Comment by XhmikosR** [comment]
And regarding the license warning, the identifier is `MPL-2.0` (hyphen), so I believe that's why you see the warning. ;)

**Comment by XhmikosR** [comment]
This can be closed after #103 is merged; the remaining ones are due to node.js 0.10 so after switching to v4 which is LTS, they should go away too.

**Comment by mozfreddyb** [comment]
:+1: thank you!
